src/wasm-lib/kcl/src/std/mod.rs

```diff
@@ -168,11 +167,35 @@ pub fn get_stdlib_fn(name: &str) -> Option<Box<dyn StdLibFn>> {
     CORE_FNS.iter().find(|f| f.name() == name).cloned()
 }
 
-pub(crate) fn std_fn(path: &str, fn_name: &str) -> crate::std::StdFn {
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct StdFnProps {
+    pub name: String,
+    pub deprecated: bool,
```

Comment:
How is this getting populated yet? I only see `default()` below that always sets it to false.

Reply:
Currently it's not because there are no deprecated functions defined in KCL, but when needed we can set this to true. I just wanted to make sure we could do this (I don't want the list of differences between Rust and KCL functions to grow)